lib/ansible/modules/cloud/azure/azure_rm_batchaccount_info.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2019 Junyi Yi (@JunyiYi)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
```

Comment:
pls also add bug fixing instruction for those auto-generated code

Reply:
@yungezz What kind of instruction would you like to add?

Reply:
2 type of info:
- the link to code generator repo
- link to instruction on bug fixing. like google module
```
#     Please read more about how to change this file at
#     https://www.github.com/GoogleCloudPlatform/magic-modules
```

Reply:
Sure thing.